man/docker-exec.1.md

```diff
@@ -7,6 +7,7 @@ docker-exec - Run a command in a running container
 # SYNOPSIS
 **docker exec**
 [**-d**|**--detach**]
+[**--detach-keys**[=*[]*]]
```

Comment:
@vdemeester is the `=` required or not...I didn't see it on the commands or did I?